stack/dashboard/base/builder.sh

```diff
@@ -14,7 +14,7 @@ set -e
 architecture="$1"
 revision="$2"
 future="$3"
-reference="$4"
+reference="$5"
```

Comment:
and `$4`?

Reply:
Done in [37aaf0d](https://github.com/wazuh/wazuh-packages/pull/1904/commits/37aaf0d5a78129b37a4aef680fa28d87079887a9)